docs/about/index.rst

```diff
@@ -8,6 +8,29 @@ About
 Mission
 -------
 
+Fairlearn is an open source toolkit that seeks to empower data scientists and
+developers to assess and improve fairness of their AI systems.
+The toolkit aspires to include:
+
+- Fairness metrics and unfairness mitigation algorithms
+- Interactive dashboard for fairness assessment
+- Educational materials (comprehensive user guide, detailed case studies,
+  white papers, etc.)
+
+Development of Fairlearn is firmly grounded in the understanding that fairness
+in AI systems is a sociotechnical challenge.  Because there are many complex
```

Comment:
Suggest we keep to starting sentences on new lines.

Reply:
done